pytgcalls/pytgcalls/group_call_native.py

```diff
@@ -283,6 +283,7 @@ async def get_group_call(self, group: Union[str, int, InputPeerChannel, InputPee
             raise RuntimeError(f'Can\'t get full chat by {group}')
 
         self.group_call = self.full_chat.call
+        self.join_as = self.full_chat.groupcall_default_join_as
```

Comment:
Setting the default join as to join as in another part of the code is not obvious. Do not assign a class var. write if statement when the join as is None. If not arg and not default - my peer. If not arg and default exists - default

Reply:
Can you review again?